diff --git a/bin/internal/shared.sh b/bin/internal/shared.sh
index 3532c23114..25dfcae4c7 100644
--- a/bin/internal/shared.sh
+++ b/bin/internal/shared.sh
@@ -229,8 +229,6 @@ function shared::execute() {
     exit 1
   fi
 
-  upgrade_flutter 7< "$PROG_NAME"
-
   BIN_NAME="$(basename "$PROG_NAME")"
   case "$BIN_NAME" in
     flutter*)
diff --git a/packages/flutter_tools/lib/src/cache.dart b/packages/flutter_tools/lib/src/cache.dart
index 13efbde879..467b3a7cbf 100644
--- a/packages/flutter_tools/lib/src/cache.dart
+++ b/packages/flutter_tools/lib/src/cache.dart
@@ -713,6 +713,7 @@ class Cache {
 
   /// Update the cache to contain all `requiredArtifacts`.
   Future<void> updateAll(Set<DevelopmentArtifact> requiredArtifacts, {bool offline = false}) async {
+    return;
     if (!_lockEnabled) {
       return;
     }
